lib/rummager_finder_publisher.rb

```diff
@@ -31,6 +31,9 @@ def pre_production?(schema)
 
   def export_finder(schema)
     presenter = FinderRummagerPresenter.new(schema[:file], schema[:timestamp])
-    RummagerWorker.perform_async(presenter.type, presenter.id, presenter.to_json)
+
+    logger.info("Publishing '#{schema[:file]['name']}' finder")
+
+    Services.rummager.add_document(presenter.type, presenter.id, presenter.to_json)
```

Comment:
That doesn't look to be the same, `rummager.add_document` is not an asynchronous operation https://github.com/alphagov/gds-api-adapters/blob/master/lib/gds_api/rummager.rb#L34 (unless I'm missing something)